Author: Pino Toscano <pino@debian.org>
Description: Use full installation paths when creating symlinks.
 Otherwise, setting the full path for either the bindir or the datadir will
 result in a path with the install prefix unconditionally added at the
 beginning.
Forwarded: no
Last-Update: 2020-08-03

--- a/cmake-modules/KDEPython.cmake
+++ b/cmake-modules/KDEPython.cmake
@@ -105,8 +105,8 @@ MACRO(KDE_ADD_PYTHON_EXECUTABLE _pyname
          MESSAGE(STATUS "Project name is necessary to create symlink against python program!!! It will fail.")
     endif(NOT PROJECT_NAME)
 
-    set(TARGET ${CMAKE_INSTALL_PREFIX}/${DATA_INSTALL_DIR}/${PROJECT_NAME}/${_pyname})
-    set(LINK_NAME ${CMAKE_INSTALL_PREFIX}/${BIN_INSTALL_DIR}/${_exename})
+    set(TARGET ${KDE_INSTALL_FULL_DATADIR}/${PROJECT_NAME}/${_pyname})
+    set(LINK_NAME ${KDE_INSTALL_FULL_BINDIR}/${_exename})
 
     GET_FILENAME_COMPONENT(abs_link_name ${LINK_NAME} ABSOLUTE)
     GET_FILENAME_COMPONENT(link_path ${LINK_NAME} PATH)
